src/plugins/validate-semantic/validators/oas3/refs.js

```diff
@@ -29,3 +30,36 @@ export const validateOAS3RefsForRequestBodiesReferenceRequestBodyPositions = ()
       }, [])
     })
 }
+
+  
+export const validateOAS3RequestBodyRefsReferenceAllowableSchemaPositions = () => sys => {
+  return sys.validateSelectors
+    .allOAS3RequestBodySchemas()
+    .then(nodes => {
+      return nodes.reduce((acc, node) => {
+        const value = node.node
+        const ref = value.$ref
+
+        if(!ref) {
+          return acc
+        }
+
+        const [, refPath = ""] = ref.split("#")
+        const pathArr = refPath.split("/") || []
+        const parentRefKey = pathArr.slice(-2)[0]
+        const targetRefKey = pathArr.slice(-1)[0]
+        if(
+          targetRefKey !== "schema"
+          && parentRefKey !== "schemas"
+          && ref.startsWith("#/")
+        ) {
+          acc.push({
+            level: "error",
+            message: `requestBody schemas $refs must point to a position where a schema can be legally placed`,
```

Comment:
**change request:** let's sharpen up the messaging here, and update the test cases to match! 🙂 

```suggestion
            message: `requestBody $ref values must point to a position where a Schema Object can be legally placed`,
```

Reply:
@shockey the 2nd replacement ("schema" -> "Schema Object") is ok, but I disagree about the 1st replacement.

"requestBody schema $refs" != "requestBody $refs". To me, the former refers to `requestBody.content.<media-type>.schema.$ref` whereas the latter refers to `requestBody.$ref`.

I would leave the 1st part of the message as is, except change
requestBody schema**s** $refs
->
requestBody schema $refs
(i.e. singular "schema")

Thoughts/objections?

Reply:
Done! Used: requestBody schema $refs and Schema Object.